presto-cli/src/test/java/io/prestosql/cli/TestClientOptions.java

```diff
@@ -98,6 +98,16 @@ public void testResourceEstimates()
                 new ClientResourceEstimate("resource2", "2.2h")));
     }
 
+    @Test
```

Comment:
what if user add two credentials with the same key and different values?

Reply:
I think his comment was simply that we should have validation for duplicates keys and fail in that case.

Reply:
Electrum is right. However I like the "namespace" concept. To me it could be just an convention in the connector itself, hence no code would need to be changed here (if we only accept that catalog could see other catalog extra credentials).

Anyway, it could go with subsequent PR.

Reply:
The duplicate keys are not allowed in `ImmutableMap::Builder`:

```    /**
     * Associates {@code key} with {@code value} in the built map. Duplicate keys are not allowed,
     * and will cause {@link #build} to fail.
     */```

I'll add a new test for duplicate credential keys.